xdocs/usermanual/jmeter_tutorial.xml

```diff
@@ -110,12 +110,27 @@ The folders inside of <code>src</code>
 </p>
 
 <dl>
+<!--
+  <dt><code>bom</code></dt><dd></dd>
+-->
+  <dt><code>bshclient</code></dt><dd>code for the BeanShell based client</dd>
   <dt><code>components</code></dt><dd>contains non-protocol-specific components like visualizers, assertions, etc.</dd>
+<!--
+  <dt><code>config</code></dt><dd>XXX</dd>
+-->
   <dt><code>core</code></dt><dd>the core code of JMeter including all core interfaces and abstract classes.</dd>
+  <dt><code>dist</code></dt><dd>build artifacts will be copied here</dd>
+  <dt><code>dist-check</code></dt><dd>code related to testing the distribution</dd>
   <dt><code>examples</code></dt><dd>example sampler demonstrating how to use the new bean framework</dd>
   <dt><code>functions</code></dt><dd>standard functions used by all components</dd>
+  <dt><code>generator</code></dt><dd>code to generate a test plan with all elements. Used for testing the distribution</dd>
   <dt><code>jorphan</code></dt><dd>utility classes providing common utility functions</dd>
+  <dt><code>launcher</code></dt><dd>code to help start and stop JMeter through API</dd>
+  <dt><code>licenses</code></dt><dd>contains information about the licenses used in JMeters dependencies</dd>
   <dt><code>protocol</code></dt><dd>contains the different protocols JMeter supports</dd>
+  <dt><code>release</code></dt><dd>XXX</dd>
+  <dt><code>testkit</code></dt><dd>utility code for testing</dd>
+  <dt><code>testkit-wiremock</code></dt><dd>utility code for testing with wiremock</dd>
```

Comment:
WireMock

Reply:
Hm, release is not described yet :(

Reply:
Do you mean https://cwiki.apache.org/confluence/display/JMETER/ReleaseCreation ?
Of course, it has to be updated for Git, however, it would probably be easier to just try Gradle and update the steps as it goes.

Reply:
I meant the `XXX` in the highlighted code above.

Reply:
oh, I see now.
I wonder if we want to split structure somehow.

For instance, so code-like modules become separated from testing/packaging ones.

It is not a complete showstopper, but things like `skipMavenPublication` in https://github.com/apache/jmeter/blob/05652933634972763c0a2af4031f64da979b78ca/src/build.gradle.kts#L19 are a bit messy.